xds/src/main/java/io/grpc/xds/internal/security/SecurityProtocolNegotiators.java

```diff
@@ -215,21 +215,16 @@ public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
       this.sslContextProviderSupplier = sslContextProviderSupplier;
       EnvoyServerProtoData.BaseTlsContext tlsContext = sslContextProviderSupplier.getTlsContext();
       UpstreamTlsContext upstreamTlsContext = ((UpstreamTlsContext) tlsContext);
-      if (CertificateUtils.isXdsSniEnabled) {
```

Comment:
How is this being removed without also changing the default value of/deleting GRPC_EXPERIMENTAL_XDS_SNI?